ieeeuoft/hackathon-template
# IEEE 63: Minimum and Maximum team size for orders

## Overview

- Resolves #[IEEE 63](https://ieeeuoft.atlassian.net/jira/software/projects/IEEE/boards/1?selectedIssue=IEEE-63)
- Enforce a minimum and maximum team size to be able to place orders


## Unit Tests Created

- Tested case where the team was smaller than min
- Tested case where the team was larger than max
- Tested case where team was between 2-4


## Steps to QA

- Using the front end, one can add item to cart
- Go to cart and try to checkout
- If team has less than 1 or more than 4 a message will appear
